etc/cdn-log-shipper/log-shipper.yml

```diff
@@ -119,6 +120,16 @@ Resources:
           });
         };
 
+        const findIp = (xff, ip) => {
+          if (xff === '-') {
```

Comment:
This is the bug that made all the hashed ips the same - the xff is coming through most of the time as `-`, which is not blank, but also not an ip. This dash was being used as the ip instead of the client ip

Reply:
Oof - good catch.  I think I had similar in the counts-lambda, but apparently forgot about it here.